tests/suites/test_suite_psa_crypto.function

```diff
@@ -4181,6 +4181,8 @@ void aead_multipart_setup( int key_type_arg, data_t *key_data,
     PSA_ASSERT( psa_import_key( &attributes, key_data->x, key_data->len,
                                 &key ) );
 
+    operation = psa_aead_operation_init( );
```

Comment:
That's not wrong, but initializing with
```
psa_aead_operation_t operation = PSA_AEAD_OPERATION_INIT;
```
would make the code easier to read. Goes everywhere this idiom is used.